Dockerfile

```diff
@@ -76,3 +76,9 @@ RUN pushd dali/python/tf_plugin/ && \
     python setup.py sdist && \
     mv dist/nvidia-dali-tf-plugin*.tar.gz /wheelhouse/ && \
     popd
+
+RUN pushd .. && \
```

Comment:
Isn't this more suited for `build docs` stage?
Especially, as our CI will build this for every python version we support.

Reply:
Created a separate task for it (DALI-787)